extensions/positron-r/package.json

```diff
@@ -654,5 +660,8 @@
     },
     "minimumRVersion": "4.2.0",
     "minimumRenvVersion": "1.0.9"
-  }
+  },
+  "enabledApiProposals": [
+    "terminalDataWriteEvent"
+  ]
```

Comment:
Will go away. Purely a development convenience, when convincing myself that we could see OSC 8 hyperlinks.